go-controller/pkg/ovn/controller/admin_network_policy/admin_network_policy.go

```diff
@@ -482,8 +481,10 @@ func (c *Controller) updateExistingANP(currentANPState, desiredANPState *adminNe
 	// Did ANP.Spec.Ingress rules get updated?
 	// (at this stage the length of ANP.Spec.Ingress hasn't changed, so individual rules either got updated at their values or positions are switched)
 	// The fields that we care about for rebuilding ACLs are
-	// (i) `ports` (ii) `actions` (iii) priority for a given rule
-	// The changes to peer labels, peer pod label updates, namespace label updates etc can be inferred
+	// (i) `ports` (ii) `actions` (iii) precedence(ACL priority) for a given rule
+	// If (i)||&& (ii), atLeastOneRuleUpdated=true already
+	// Next let's check for a peer update:
```

Comment:
I don't understand these changes

Reply:
hmm me neither I guess I was drunk or something. clearly what I wrote on L484 is wrong it is priority not precedence.

Reply:
reverting this change, its not making sense..